scripts/tests/api_compare/filter-list-offline

```diff
@@ -32,3 +32,6 @@
 !Filecoin.EthGetBlockByNumber
 !eth_getBlockByNumber
 !Filecoin.ChainSetHead
+!Filecoin.EthFeeHistory
```

Comment:
Could you add some comments on why these are filtered out?

Reply:
these are now removed from filter list as i have changed test type to `basic` because to compare methods that take predefined block string(like `pending`, `latest`, `safe` or `finalized`) chain head for both the comparing nodes must be in sync if not you get different tipsets to compare which will fail eventually